environment.yml

```diff
@@ -2,12 +2,17 @@ name: pyleco
 channels:
   - conda-forge
 dependencies:
-  - pyzmq=25.0.0
+  - pyzmq=25.1.0
+  - pip  # don't pin, to gain newest conda compatibility fixes
+  - pip:
+    - jsonrpc2-pyclient==2.2.12
+    - openrpc==6.3.17
+    - uuid7==0.1.0
 # Development dependencies below
   - pytest=7.2.0
+  - pytest-cov=4.1.0
+  - pytest-runner=6.0.0
   - setuptools_scm # don't pin, to get newest features
-  - sphinx=6.1.3
-  - sphinx_rtd_theme=1.0.0
-  # - pip  # don't pin, to gain newest conda compatibility fixes
-  # - pip:
-  #   - xxx
+  - pip:
+    - sphinx==6.1.3
+    - sphinx_rtd_theme==1.2.2
```

Comment:
hm? Those should be available from conda, no problem?

Reply:
Conda did tell me, that it could not find them, even though they are in conda forge 🤷

Reply:
that can't be right... maybe a weird packaging constellation, we should troubleshoot that.
```
# Name                       Version           Build  Channel             
 sphinx_rtd_theme               1.2.2    pyha770c72_0  conda-forge   

# Name                       Version           Build  Channel             
sphinx                         6.1.3    pyhd8ed1ab_0  conda-forge   
```

Reply:
Yes, they are in coda forge, but somehow, Conda told me, that it could not find the.

Reply:
Ok, then I'll need a more detailed repro procedure and error messages to diagnose this. OS? On CI or your personal device, or both?

Reply:
Only CI. I pushed a commit, which uses Conda forge instead of pip and which should fail in order to produce that log.

Reply:
We need an older sphinx version, it seems, to be compatible with sphinx rtd theme

Reply:
With another version (sphinx 5), it works.